teams/teams-ps/teams/New-CsTeamsCallingPolicy.md

```diff
@@ -568,6 +593,42 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -EnableSpendLimits
+This setting allows an admin to enable or disable spend limits on PSTN calls for their user base.
+
+Possible values: 
+- True
+- False
+
+```yaml
+Type: Boolean
+Parameter Sets: (All)
+Aliases:
+Applicable: Microsoft Teams
+Required: False
+Position: Named
+Default value: None
```

Comment:
Please add information on the default value for the parameter.

Reply:
Updated. Added 'False' as the default value.